policy/handler.go

```diff
@@ -102,7 +102,14 @@ func (h *Handler) Run(ctx context.Context, evalCh chan<- *Evaluation) {
 			return
 		case <-h.doneCh:
 			return
+
 		case err := <-h.errCh:
+			// In case of error, log the error message and loop around.
+			// Handlers never stop running unless ctx.Done() or doneCh is
+			// closed.
+
+			// multierror.Error objects are logged differently to allow for a
```

Comment:
if we have to do this somewhere else in the future, we should certainly pull the logic into a helper.

Reply:
Yeah, it might be worth to even add this to `hclog`. I will keep it here for now since policy validation is the only place we use `multierror` so far.